sos/plugins/cifs.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Cifs(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """SMB file system information
+    """
+    plugin_name = 'cifs'
+    profiles = ('storage', 'network', 'cifs')
+    packages = ['cifs-utils']
+
```

Comment:
Might as well be a tuple here. It's never going to benefit from any of the list features.